### ISSUE TITLE
Wireframe Experimental feature suggestions

### ISSUE THREAD

**Issue by LouisDraw** [ArmoredColony/ARMORED-Toolkit#3]
Hi, I just have some suggestions about the wireframe experimental feature : 

- I love it, but sometimes, I have to disable it to see clearly my scene. So maybe it should be very cool to **get a button** or something (maybe in the left off the header ?) so we can use it whenever it's useful (and add it to our quick favorite).
- and when we switch the mode to the edit one, the wireframe is still there, and it causes some issues to read the topology (when sub d is enabled mostly) it would be cool that the **wireframe automatically turn off when we switch the edit mode** and go back to the object mode.

Thanks a lot for this addon, it's really helpful !! (sorry for the english mistakes)

**Comment by ArmoredColony** [comment]
I like that idea. 

Also, it might not always work, hence the Experimental tag, which is a nice way of saying I didn't finish it, but I'll take another look since at least one person wants to use it.

**Comment by LouisDraw** [comment]
Yeah i know its an experimental feature but its a good one in my opinion. Thank you !!

**Comment by ArmoredColony** [comment]
I couldn't figure out a good way to implement this consistently. I wouldn't be surprised if there is a way to do this with some bpy handler, but it is still beyond my skills for now.

**Comment by ArmoredColony** [comment]
Wireframe Selected seems to be working okay now (v085). It can only be toggled on/off from the addon preferences for now. 

I still need to find another good UI location for this toggle (to make it more accessible), but its working.
